modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,23 @@
+= B2B Solutions
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: b2b, edi, portal
+
+== B2B (Business-to-Business)
+
+B2B in MuleSoft is the use of API's or Electronic Data Interchange (EDI) by enterprises to carry out business transactions such as purchasing goods from each other. B2B messages are typically sent over transports such as SFTP and exchanges such as AS2. MuleSoft's B2B solutions integrate seamlessly into the Mule runtime.
```

Comment:
I think "API's" should be changed to "APIs", if it is referring to API in plural. Not completely sure about this though.

Reply:
You are right.